adage/analyze/models.py

```diff
@@ -173,7 +173,7 @@ class Activity(models.Model):
     value = models.FloatField()
 
     def __unicode__(self):
-        return "Sample %s at Node %s with value %f" % (self.sample.sample,
+        return "Sample %s at Node %s with value %f" % (self.sample.name,
```

Comment:
Nice catch. The old way with `.sample` should work, but give the result from calling `__unicode__`, I believe.